README.md

```diff
@@ -315,9 +315,9 @@ Cipher used: AES-GCM with 256 bit keys.
 
 ## Obligatory Disclaimer About Passwords
 
-🚨 **DO NOT USE THIS TO ENCRYPT PASSWORDS** 🚨
+🚨 **DO NOT USE THIS TO ENCRYPT PASSWORDS WITHOUT ADDITIONAL SECURITY MEASURES** 🚨
 
-Passwords should be hashed & salted using a slow, constant-time one-way function.
+Passwords should be hashed & salted using a slow, constant-time one-way function. This library may only be used to encrypt the salted and hashed password as a [pepper](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#peppering) to provide an additional layer of security. It is recommended that the encryption key be stored in a [Hardware Security Module](https://en.wikipedia.org/wiki/Hardware_security_module) on the server.
```

Comment:
Could you rephrase "This library may only be used" to something like "However, this library could be used"?

It could be ambiguous that this is the sole purpose of the library otherwise.